### DISCUSSION TITLE
Location for recording videos (NB: not yet available)

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#69] [Team Posts]
Our idea for posting the recordings is to put them up on YouTube as unlisted and then post the links to the discussions here for the `participants' group --- and only leave them up for a fixed amount of time (e.g. 2 months).

The open question right now is whether these discussions are sufficiently private. If I try to go to https://github.com/orgs/delph-in/teams/participants when not loggen in, it asks me to login in to GitHub, but what I can't tell is whether just logging into GitHub with *any* GitHub account would let me see them. Can anyone with an additional GitHub account that is not part of this group verify that for me?

**Comment by olzama** [comment]
Hi Emily,

So, the question is, do GitHub users who aren’t in the participants team
see the page linked below?

On Tue, Jul 19, 2022 at 5:11 PM emilymbender ***@***.***>
wrote:

> Our idea for posting the recordings is to put them up on YouTube as
> unlisted and then post the links to the discussions here for the
> `participants' group --- and only leave them up for a fixed amount of time
> (e.g. 2 months).
>
> The open question right now is whether these discussions are sufficiently
> private. If I try to go to
> https://github.com/orgs/delph-in/teams/participants when not loggen in,
> it asks me to login in to GitHub, but what I can't tell is whether just
> logging into GitHub with *any* GitHub account would let me see them. Can
> anyone with an additional GitHub account that is not part of this group
> verify that for me?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/teams/participants/discussions/20>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACTURKQUHADW2ELEOVQXKP3VU47ZNANCNFSM54BW3ALA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Olga

**Comment by ltxom** [comment]
Hi Emily, 

I tried with another GitHub account (which is not in Delph-in group), and accessing this discussions page (https://github.com/orgs/delph-in/teams/participants) shows a 404 error.

**Comment by emilymbender** [comment]
Thanks, @ltxom ! So it seems that we can safely post unlisted links here and thus make the videos only available to DELPH-IN participants.

@sweaglesw -- can you share the recordings you have with me? I will post them as unlisted YouTube videos, share the links here, and then make a note to take them down in 2 months.
